modules/nf-core/ascat/main.nf

```diff
@@ -40,29 +40,29 @@ process ASCAT {
     def gc_input       = gc_file              ?  "$gc_file" :            "NULL"
     def rt_input       = rt_file              ?  "$rt_file" :            "NULL"
 
-    def minCounts_arg                    = args.minCounts                     ?  ",minCounts = $args.minCounts" : ""
-    def bed_file_arg                     = bed_file                           ?  ",BED_file = '$bed_file'": ""
-    def chrom_names_arg                  = args.chrom_names                   ?  ",chrom_names = $args.chrom_names" : ""
-    def min_base_qual_arg                = args.min_base_qual                 ?  ",min_base_qual = $args.min_base_qual" : ""
-    def min_map_qual_arg                 = args.min_map_qual                  ?  ",min_map_qual = $args.min_map_qual" : ""
-    def fasta_arg                        = fasta                              ?  ",ref.fasta = '$fasta'" : ""
-    def skip_allele_counting_tumour_arg  = args.skip_allele_counting_tumour   ?  ",skip_allele_counting_tumour = $args.skip_allele_counting_tumour" : ""
-    def skip_allele_counting_normal_arg  = args.skip_allele_counting_normal   ?  ",skip_allele_counting_normal = $args.skip_allele_counting_normal" : ""
+    def minCounts_arg                    = args.minCounts                     ?  ", minCounts = $args.minCounts" : ""
+    def bed_file_arg                     = bed_file                           ?  ", BED_file = '$bed_file'": ""
+    def chrom_names_arg                  = args.chrom_names                   ?  ", chrom_names = $args.chrom_names" : ""
+    def min_base_qual_arg                = args.min_base_qual                 ?  ", min_base_qual = $args.min_base_qual" : ""
+    def min_map_qual_arg                 = args.min_map_qual                  ?  ", min_map_qual = $args.min_map_qual" : ""
+    def fasta_arg                        = fasta                              ?  ", ref.fasta = '$fasta'" : ""
+    def skip_allele_counting_tumour_arg  = args.skip_allele_counting_tumour   ?  ", skip_allele_counting_tumour = $args.skip_allele_counting_tumour" : ""
+    def skip_allele_counting_normal_arg  = args.skip_allele_counting_normal   ?  ", skip_allele_counting_normal = $args.skip_allele_counting_normal" : ""
 
     """
     #!/usr/bin/env Rscript
     library(RColorBrewer)
     library(ASCAT)
     options(bitmapType='cairo')
 
-    #build prefixes: <abspath_to_files/prefix_chr>
-    allele_path = normalizePath("$allele_files")
-    allele_prefix = paste0(allele_path, "/", "$allele_files", "_chr")
+    # Build prefixes: <abspath_to_files/prefix_chr>
+    allele_path = basename(normalizePath("$allele_files"))
+    allele_prefix = sub('_chr[0-9]+\\\\.txt\$', "_chr", allele_path)
 
-    loci_path = normalizePath("$loci_files")
-    loci_prefix = paste0(loci_path, "/", "$loci_files", "_chr")
+    loci_path =  basename(normalizePath("$loci_files"))
+    loci_prefix = sub('_chr[0-9]+\\\\.txt\$', "_chr", loci_path)
```

Comment:
@LouisLeNezet , any idea why you made this change?
This is breaking the use of ASCAT on sarek.